From b5c7a144ab0adb676d1909ff9f24456c4f13781e Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Wed, 3 Apr 2019 18:49:08 +0000
Subject: [PATCH] TOOLS-2233 mailing lists have been migrated from Listbox to
 Topicbox Reviewed by: Isaac Davis <isaac.davis@joyent.com> Approved by: Isaac
 Davis <isaac.davis@joyent.com>

---
 README.md | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/README.md b/README.md
index d3d0693..6ed9f89 100644
--- a/README.md
+++ b/README.md
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2017, Joyent, Inc.
+    Copyright 2019, Joyent, Inc.
 -->
 
 # Manta: Triton's object storage and converged analytics solution
@@ -62,11 +62,10 @@ see [Dave Pacheco](https://github.com/davepacheco)'s blog entry on
 
 Community discussion about Manta happens in two main places:
 
-* The *manta-discuss* mailing list.  Once you [subscribe to the
-  list](https://www.listbox.com/subscribe/?list_id=247448), you can send mail
-  to the list address: manta-discuss@lists.mantastorage.org.  The mailing list
-  archives are also [available on the
-  web](https://www.listbox.com/member/archive/247448/=now).
+* The *manta-discuss*
+  [mailing list](https://mantastorage.topicbox.com/groups/manta-discuss).
+  If you wish to send mail to the list you'll need to join, but you can view
+  and search the archives online without being a member.
 
 * In the *#manta* IRC channel on the [Freenode IRC
   network](https://freenode.net/).
-- 
2.21.0

